### ISSUE TITLE
Haversine distance macro doesn't support kms

### ISSUE THREAD

**Issue by joellabes** [dbt-labs/dbt-utils#338]
### Describe the bug
The macro to calculate the haversine distance between two points always returns its result in miles, which isn't useful for people who use the metric system

### Steps to reproduce
- Work somewhere other than [Liberia, Myanmar, the United Kingdom and the United States](https://en.wikipedia.org/wiki/Mile#International_mile)
- Use the haversine_distance macro

### Expected results
- The option to get results in either miles or kilometres (or even kilometers)

### Actual results
- Results in miles

### System information
**The contents of your `packages.yml` file:**
```yaml
packages:
  - package: fishtown-analytics/dbt_utils
    version: 0.6.3
  
  - package: fishtown-analytics/audit_helper
    version: 0.3.0

  - package: fishtown-analytics/codegen
    version: 0.3.1

  - package: fishtown-analytics/redshift
    version: 0.4.1
```
**Which database are you using dbt with?**
- [ ] postgres
- [x] redshift
- [ ] bigquery
- [ ] snowflake
- [ ] other (specify: ____________)

**The output of `dbt --version`:**
```
0.19.0 (dbt Cloud)
```

### Additional context
The [great circle calculator](https://andrew.hedges.name/experiments/haversine/) linked from the original macro's source says to use 6373km as the radius of the Earth, which is as good a number as any

### Are you interested in contributing the fix?
Yes, but not for a while.

**Comment by bastienboutonnet** [comment]
We've taken an approach on this to just convert the results from that macro and divide it by `1.60934`

May not be super exact but since we're already in the realm of non exact things that might work out.

I'd be happy to PR the haversine data macro and maybe make it take a param for `km` or `mi`

**Comment by joellabes** [comment]
Resolved by #340!
